lib/src/logic.dart

```diff
@@ -575,30 +575,88 @@ class Logic {
   /// Greater-than-or-equal-to.
   Logic operator >=(dynamic other) => GreaterThanOrEqual(this, other).out;
 
-  /// Pre-Increment (++var) or var+=incrVal
-  // ignore: prefer_expression_function_bodies
-  ConditionalAssign preIncr({Logic? incrVal}) {
-    return this < ((incrVal != null) ? this + incrVal : this + Const(1));
-  }
+  /// Shorthand Increment.
```

Comment:
Let's elaborate a bit more in the doc comments.  Something like "Shorthand for a [Conditional] which increments `this` by [incrVal]".  Make sure the first line of the doc comment has a blank line afterwards.
```dart
/// Short descriptions
///
/// Further details down here
void myFunction...
```

In general, for these doc strings, try to make them full english sentences that clearly explain the purpose of the API.